### PR TITLE
Ecs dev

### PR DESCRIPTION
**Pull request by aaronFranssell**
### JIRA Ticket

SDC-1895

### Summary

Initial script to get webportal deployments working in dev ECS.

### CODE REVIEW

File: webportal/lambda/deployment/deploy_lambda.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 def deploy_backend(command_line_arguments):
     parsed_arguments = cli_parser.parse_arguments(command_line_arguments)
-    chalice_deploy_command = f'chalice deploy --stage {parsed_arguments.environment} --no-autogen-policy --profile sdc'
+    chalice_deploy_command = f'chalice deploy --stage {parsed_arguments.environment} --no-autogen-policy'
+    if not parsed_arguments.environment.startswith('ecs'):
+        chalice_deploy_command += ' --profile sdc'
```
Comment (JAMSUPREME) [inline webportal/lambda/deployment/deploy_lambda.py:9]: I was expecting `--profile ecs` or something like that. How are your creds being pulled?

File: webportal/lambda/setup_venv.sh [hunk 2]
```diff
@@ -2,4 +2,13 @@
...
 . .venv/bin/activate
-pip3 install -r requirements.txt -r development_requirements.txt
\ No newline at end of file
+pip3 install -r requirements.txt -r development_requirements.txt
+
+if test -f "/proc/sys/crypto/fips_enabled"; then
+    echo "On FIPS compliant system, executing ugly, ugly hack."
+    # For more details on this monstrosity, see this issue here:
+    # https://github.com/aws/chalice/issues/1442
+    sed -i 's/hashlib.md5(contents)/hashlib.md5(contents,usedforsecurity=False)/g' .venv/lib64/python3.6/site-packages/chalice/deploy/packager.py
```
Comment (JAMSUPREME) [inline webportal/lambda/setup_venv.sh:11]: Lol that's something... 🤣

File: webportal/lambda/deployment/deploy_lambda.py (hunk 1 above)
Comment (aaronFranssell) [reply webportal/lambda/deployment/deploy_lambda.py:9]: These are being run on an EC2 RHEL instance with a PowerUser profile. I'm just pretending I'm Jenkins running the Terraforms...

File: webportal/lambda/setup_venv.sh (hunk 2 above)
Comment (aaronFranssell) [reply webportal/lambda/setup_venv.sh:11]: It's a mighty hack indeed...

File: webportal/lambda/tests/deployment/test_gateway_normalizer.py [hunk 3]
```diff
@@ -106,4 +107,26 @@ def test_normalize_sets_vpce(mock_chalice_config_reader, partially_mocked_gatewa
...
+        chalice_config_reader.chalice_config.assert_called_with()
+
+
+def test_get_session_with_fips_enabled_returns_session_with_default_profile(partially_mocked_gateway_normalizer):
+    result = partially_mocked_gateway_normalizer.get_session('tests/fixtures/some_config.json')
+
+    assert result.profile_name == 'default'
+
+
+def test_get_region_returns_region():
```
Comment (aaronFranssell) [inline webportal/lambda/tests/deployment/test_gateway_normalizer.py:119]: Lot of work to get above 90% coverage...

**Pull request merged**